<!--
roslaunch autoware_camera_lidar_calibrator evaluate_camera_lidar_calibration.launch calib_param_yaml:=/home/zhenzhen/saic_ws/src/autoware_camera_lidar_calibrator/autoware_camera_lidar_calibrator/data/20191209_190748_autoware_lidar_camera_calibration.yaml
-->
<launch>
    <arg name="image_src" default="/image_raw" />
    <arg name="camera_info_src" default="/camera_info" />
    <arg name="camera_id" default="/" />
    <arg name="calib_param_yaml" />
    <arg name="points_src" default="/points_raw" />

    <node pkg="autoware_camera_lidar_calibrator" type="calibration_publisher" name="calibration_publisher" args="$(arg calib_param_yaml)" ns="$(arg camera_id)">
        <param name="register_lidar2camera_tf" type="bool" value="true"/>
        <param name="publish_extrinsic_mat" type="bool" value="true"/>
        <param name="publish_camera_info" type="bool" value="true"/>
        <param name="target_frame" type="string" value="velodyne_points"/>
    </node>

    <node pkg="points2image" type="points2image" name="points2image">
        <param name="points_node" type="string" value="$(arg points_src)"/>
    </node>

    <node pkg="points2image" type="points_image_viewer" name="points_image_viewer">
        <param name="image_raw_topic" type="string" value="$(arg image_src)"/>
    </node>

</launch>
